packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -254,7 +254,7 @@ export default declare((api, options, dirname) => {
         if (cached) {
           cached = t.cloneNode(cached);
         } else {
-          cached = addDefault(file.path, source, {
+          cached = addDefault(file.path, `${source}.js`, {
```

Comment:
Nit: I would prefer not to add the extension here and to add it where `addDefaultImport`, so that we are explicit about the files we are importing with `assDefaultImport`.

Reply:
@nicolo-ribaudo 
Is it fine c3b1bf5?